src/Components/Details/ShowDetails.js

```diff
@@ -20,25 +20,24 @@ class ShowDetails extends Component {
     const taintedRating = '⭐️';
     const accessDate = specificMovie.release_date ? specificMovie.release_date.split('-')[0] : '';
     const taglineConditional = specificMovie.tagline ? specificMovie.tagline : 'Write in and give us a tagline for this one!';
-    const imageAltTag = `movie poster for ${specificMovie.title}`
 
         return (
           <div className='details-card'>
-            <NavLink exact to='/'><button className='home-button' onClick={() => handleClick(false)}>Back to Home Page</button></NavLink>
+            <NavLink exact to='/'><button data-cy='home-button' className='home-button' onClick={() => handleClick(false)}>Back to Home Page</button></NavLink>
             {this.state.error && <h3>{this.state.error}</h3>}
             {!this.state.error &&
               <>
-                <img className='details-image' src={specificMovie.poster_path} alt={imageAltTag}/>
-                <h2 className='details-title'>{specificMovie.title}</h2>
-                <h4 className='movie-release'>{accessDate}</h4>
-                <h4 className='movie-rating'>{taintedRating.repeat(Math.floor(specificMovie.average_rating))}</h4>
-                <h4 className='tagline'>{`'${taglineConditional}'`}</h4>
-                <h4 className='overview'>Movie Overview:</h4>
-                <h4 className='overview'>{specificMovie.overview}</h4>
-                <h4 className='genre'>Genre: {specificMovie.genres}</h4>
-                <h4 className='budget'>Budget: ${specificMovie.budget}</h4>
-                <h4 className='revenue'>Revenue: ${specificMovie.revenue}</h4>
-                <h4 className='runtime'>RunTime: {specificMovie.runtime} min</h4>
+                <img className='details-image' src={specificMovie.poster_path} alt={`movie poster for ${specificMovie.title}`}/>
```

Comment:
I will review previous branches to verify all changes are here to stay.

Here we want and have previously committed to less variables.